lib/src/repository/traits/thermostat_repository.dart

```diff
@@ -93,6 +99,23 @@ class AvailableFanModes extends Property<Set<AvailableFanMode>> {
       : super('availableFanModes', value);
 }
 
+class TemperatureRange {
+  final double min;
```

Comment:
Should we set these to `final double? min` and start supporting nullable fields for this and other state values?

Reply:
As discussed we will follow the graphql schema and these min and max are marked as required value over there.